services/blockchain-service/blockchain-service-base.js

```diff
@@ -49,36 +49,39 @@ class BlockchainServiceBase {
         return;
     }
 
+    async getNetworkGasPrice(blockchain) {
+        const web3Instance = await this.getWeb3Instance(blockchain);
+
+        try {
+            return await web3Instance.eth.getGasPrice();
+        } catch (error) {
+            console.error(`Failed to fetch the gas price from the network: ${error}. Using default value: 100 Gwei.`);
```

Comment:
perhaps we should use console.info or console.warn instead of console.error here